tests/test_dataIdContainer.py

```diff
@@ -0,0 +1,66 @@
+# This file is part of pipe_base.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
```

Comment:
Please add the COPYRIGHT file (or change this not to reference it)

Reply:
@timj came up with a list of things to put in that file. I don't think that process will scale for any other packages that we add new files to...